README.md

```diff
@@ -210,6 +210,18 @@ This will copy the `hosts.yaml` configuration file from
 that it is picked up by Pennyworth automatically. This makes it easy to
 distribute a common configuration between several systems and users.
 
+Pennyworth automatically cleans up the hosts after running the tests using
+snapper snapshots. The snapshot which the system is rolled back to is specified
+by the `base_snapshot_id` of the host entries. Before rolling back a new
+snapshot named `pennyworth` is created which can be used for debugging test
+failures.
+
+In some cases (e.g. while working on a test) the automatic rollback might not be
+desired. It can easily be disabled temporarily by setting the `SKIP_CLEANUP`
+environment variable, e.g.
+
+    SKIP_CLEANUP=true rspec
```

Comment:
Wouldn't it be better to use some configuration either in the code or in the config file than relying on an environment variable? Then you wouldn't need to think of providing the variable with every invocation of the test.